src/bin/cargo/cli.rs

```diff
@@ -54,10 +54,9 @@ pub fn main(config: &mut LazyConfig) -> CliResult {
         .map(String::as_str)
         == Some("help")
     {
-        let header = style::HEADER.render();
-        let literal = style::LITERAL.render();
-        let placeholder = style::PLACEHOLDER.render();
-        let reset = anstyle::Reset.render();
+        let header = style::HEADER;
```

Comment:
As a subsequent followup, might it make sense to `use style::{HEADER, LITERAL, PLACEHOLDER};`, and then use the constants directly, rather than first putting them in locals?